src/scicookie/ui.py

```diff
@@ -3,13 +3,14 @@
 from __future__ import annotations
 
 import os
+import re  # Add this import for regex
```

Comment:
you don't need this comment here .. it is really clear that `re` is used for regex among the python users
so no worries about the comment